README.md

```diff
@@ -1 +1,61 @@
-# Bookstore-react
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Bookstore with react
+
+This is a simple Bookstore web app made with react
+
+## Built With
+
+- HTML
+- CSS
+- Sass
+- Javascript
+
+## Frame work
+
+ - React js
+
+# Getting Started with Create React App
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```

Comment:
- [ ] Kindly list instructions below on how one would be able to use the app:

```mdx
git clone ....... repo name
cd projectDirectory
npm i
npm start
```
That will be very helpful in documenting your app in the Getting started section of your README 👍